Makefile

```diff
@@ -19,7 +19,7 @@ run-containers: ## run all test containers
 	@cd test_containers && docker-compose up -d && cd ..
 
 kill-containers: ## kill all test containers
-	@cd test_containers && docker-compose stop && cd ..
+	@cd test_containers && docker-compose down && cd ..
```

Comment:
You changed from `stop` to `down` because the `down` command will stop and remove the containers, right?

Reply:
Yes!